lib/money/currency.rb

```diff
@@ -26,6 +26,10 @@ def find(currency_iso)
       def currencies
         @@currencies ||= Loader.load_currencies
       end
+
+      def crypto_currencies
```

Comment:
I didn't see in the PR description if this was a deliberate choice. Another alternative would've been to extend the loader to be aware of what to load and just have one internal hash of all currencies (that is the union of fiat and crypto currencies) which would mean that other dependent code that's using `Money::Currency.currencies` would just work. A typical example for this is code that generates a schema and generates an enum of currencies.

One can argue either way but just curious. 

Regardless of this I'd document the behaviour in the new crypto section in the doc. It might be surprising for consumers of the API.